src/postgres/src/test/regress/yb_pg_foreign_data_schedule

```diff
@@ -0,0 +1,10 @@
+# src/test/regress/yb_pg_foreign_data_schedule
+#
+####################################################################################################
+# Postgres Testsuites: Porting from PostgreSQL original tests.
+# This includes tests related to postgres foreign data.
+####################################################################################################
+# Dependency for `yb_pg_foreign_data`
+test: yb_create_function
+
+test: yb_pg_foreign_data
```

Comment:
nit: newline at the end of file

Reply:
Resolved: https://github.com/yugabyte/yugabyte-db/pull/9650/commits/ce0f8924c9c49dca9ea9279912a3ac815cb35917